### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by okolenmi** [rklaffehn/rk-comfy-nodes#1]
I tried to just put it between decoder and image saver, but it doesn't work. I can see only some messages in console about hashes. I thought these nodes were designed to add resource hashes to automatically detect them on civitai. Am I right or not?

**Comment by rklaffehn** [comment]
You're right; my goal is that CivitAI eventually detects the used resources when you upload images made with ComfyUI. Adding the model's hash values to the prompt is one step toward the goal.

However, CivitAI needs a patch, too, to read the hashes. This will take a while as CivitAI is a rather complex piece, and I still need to get a local version running for tests.

**Comment by okolenmi** [comment]
![изображение](https://github.com/rklaffehn/rk-comfy-nodes/assets/55319814/5fb8fa67-aaf1-4b76-838e-066fa91c3b22)
Will it work if you make the same structure as in A1111 output images? _A text editor is not very suitable for reading metadata, but you can see how it was done._
